apps/public/src/app/(map)/locatie/kies/page.module.css

```diff
@@ -0,0 +1,41 @@
+.page {
+  grid-template-rows: auto 1fr auto;
+  height: 100vh;
+}
+
+.listView {
+  grid-row-start: 2;
+  position: relative;
+
+  @media screen and (min-width: 68rem) {
+    grid-row-start: 2;
+  }
+}
+
+.hideListView {
+  visibility: hidden;
+
+  @media screen and (min-width: 68rem) {
+    visibility: unset;
+  }
+}
+
+.mapView {
+  grid-row-start: 2;
+
+  @media screen and (min-width: 68rem) {
+    grid-row: 1 / -1;
+  }
+}
+
+.buttonColumn {
```

Comment:
I think you want the [Row component](https://designsystem.amsterdam/?path=/docs/components-layout-row--docs)